docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc

```diff
@@ -19,11 +20,11 @@ function content_type_phone_book_ctools_plugin_api($module = NULL, $api = NULL)
 function content_type_phone_book_node_info() {
   $items = array(
     'phone_book' => array(
-      'name' => t('Contact item'),
```

Comment:
It is not ok to have Danish strings in the t() function. It should be the English string and then to do the translation manually via the Translation UI after deploy

Reply:
@lachezarvalchev, fixed.